crates/uv-dev/src/resolve_many.rs

```diff
@@ -140,7 +140,11 @@ pub(crate) async fn resolve_many(args: ResolveManyArgs) -> Result<()> {
                     requirement
                 };
 
-                let result = build_dispatch.resolve(&[requirement.clone()]).await;
+                let result = build_dispatch
+                    .resolve(&[
+                        UvRequirement::from_requirement(requirement.clone()).expect("TODO(konsti)")
```

Comment:
We should resolve `.expect("TODO(konsti)")`.

Reply:
thx